app/models/vm_or_template.rb

```diff
@@ -1904,6 +1904,14 @@ def self.reconfigurable?(ids)
     vms.all?(&:reconfigurable?)
   end
 
+  def self.tenant_id_clause(user_or_group)
+    template_tenant_ids = MiqTemplate.accessible_tenant_ids(user_or_group, Rbac::TENANT_ACCESS_STRATEGY["MiqTemplate"])
+    vm_tenant_ids       = Vm.accessible_tenant_ids(user_or_group, Rbac::TENANT_ACCESS_STRATEGY["Vm"])
```

Comment:
The code above calls `Rbac.accessible_tenant_ids_strategy` but this code accesses the constant directly.  Should it also go through the accessor?